docs/source/fury-pybullet.rst

```diff
@@ -0,0 +1,606 @@
+Fury - pyBullet Integration Guide
+=================================
+
+* :ref:`Simple Rigid body dynamics <simple_rigid_body_dynamics>`
+
+  * :ref:`Necessary Imports <imports>`
+
+  * :ref:`Connection Mode <connection>`
+
+  * :ref:`Disconnection <disconnection>`
+
+  * :ref:`Setting Gravity <gravity>`
+
+  * :ref:`Creating Objects <objects>`
+
+  * :ref:`Changing Object Dynamics <change_dynamics>`
+
+  * :ref:`Adding objects to the scene <add_to_scene>`
+
+  * :ref:`Application of Force/Torque <apply_force_torque>`
+
+  * :ref:`Enabling Collision <enable_collision>`
+
+  * :ref:`Creation of Show Manager <show_manager>`
+
+  * :ref:`Syncing properties of Actors <sync_actors>`
+
+  * :ref:`Creation of timer callback <timer_callback>`
+
+  * :ref:`Initiating the simulation <init_simulation>`
+
+  * :ref:`Rendering multiple objects by a single actor <single_actor>`
+
+  * :ref:`Rendering Joints <render_joints>`
+
+
+* :ref:`Examples <examples>`
+
+  * :ref:`Brick Wall Simulation <brick_wall_multi>`
+
+  * :ref:`Ball Collision Simulation <ball_collision>`
+
+  * :ref:`Brick Wall Simulation(Single Actor) <brick_wall_single>`
+
+  * :ref:`Chain Simulation <chain_simulation>`
+
+  * :ref:`Wrecking Ball Simulation <wrecking_ball_simulation>`
+
+  * :ref:`Domino Simulation <domino_simulation>`
+
+
+**Official docs:**
+  * `FURY <https://fury.gl/latest/reference/index.html>`__
+
+  * `pyBullet <https://docs.google.com/document/d/10sXEhzFRSnvFcl3XxNGhnD4N2SedqwdAvK3dsihxVUA/edit#>`__
+
+**NOTE: All elements are in SI units.**
```

Comment:
```
.. note::
  **All elements are in SI units.**
```
instead.
more info [here](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#colored-boxes-note-seealso-todo-and-warnings)